.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
♻️ Hi @lenkon, your project is going great!! However, there are some things that you need to work on. Linters are super useful tools that allow you to find bugs even before compiling/shipping code. Right now, you have set the GitHub actions to run the linters. Good Job! Something missing is the configuration files for the linters.

- [ ] Please double-check the [linters config guide](https://github.com/microverseinc/linters-config/tree/master/html-css) and kindly add the missing files.⚙️
- [ ] Also please note that your GitHub Actions should return green when creating the pull request. Currently, the Stylelint action is returning red. Please fix the linter errors based on the information returned by the linter by running the `npx stylelint "**/*.{css,scss}"`

Please note that for future projects, these linters must be green before submitting the PR for review.